src/infra/core/src/repos/dataset_repository_local_fs.rs

```diff
@@ -517,7 +550,7 @@ impl DatasetStorageStrategy for DatasetSingleTenantStorageStrategy {
                 }
                 let dataset_name = DatasetName::try_from(&dataset_dir_entry.file_name()).int_err()?;
                 let dataset_alias = DatasetAlias::new(None, dataset_name);
-                match self.resolve_dataset_alias(&dataset_alias).await {
+                match self.attempt_resolve_dataset_alias(&dataset_alias).await {
```

Comment:
Why do we call `resolve_dataset_alias` here to begin with?

We already have the alias from the fs path, we know that directory exists. I think we can directly call `attempt_resolving_summary_via_path` just to get the ID of the dataset to construct the handle, or your helper method `resolve_dataset_handle`.

Reply:
agree, removed